### PR TITLE
Finished javascript-loops-exercise. Lambert Nguyen

### PR DESCRIPTION
**Pull request by Ln1097**
Screenshot (175)
Screenshot (183)
Screenshot (182)
Screenshot (181)

### CODE REVIEW

File: javascript-loops.tar/javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
+console.log(typeof repeatWord('cool', 10));
+console.log(repeatWord('cool', 10));
+
+function logEachCharacter(string) {
+  for (let i = 0; i < 3; i++) {
```
Comment (kurtyazdizadeh) [inline javascript-loops.tar/javascript-loops/main.js:44]: `i < 3` is not right. That would only work for strings that are 3 characters long. What if I passed in a string longer than three characters? How could I re-write that expression to dynamically handle any length of string that is passed in from the argument?

File: javascript-loops.tar/javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    console.log(string[i]);
+  }
+}
+console.log(logEachCharacter('fun'));
+
+function doubleAll(numbers) {
+  const doubled = [];
+  for (let i = 0; i < 3; i++) {
```
Comment (kurtyazdizadeh) [inline javascript-loops.tar/javascript-loops/main.js:52]: same idea here, this only works if the `numbers` array had a length of 3. It shouldn't be hardcoded like this, what property on the `numbers` array could we use instead to dynamically handle the `length` in the for loop? 🤔

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Take a look at my comments for some small changes!

**Comment by Ln1097** [comment]
Screenshot (188)

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**